tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -261,6 +264,16 @@ public void Dispose()
             _waf?.Dispose();
         }
 
+        private void AsmDDProductConfigChanged(object sender, ProductConfigChangedEventArgs e)
+        {
+            var asmDD = e.GetDeserializedConfigurations<string>().FirstOrDefault();
+            if (!string.IsNullOrEmpty(asmDD))
+            {
+                _remoteRulesJson = asmDD;
```

Comment:
does this need to be stored in the `Security` class, can't it be passed as a parameter to `UpdateStatus`?